students/dodam/westagram/users/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
+	name = models.CharField(max_length=45)
+	email = models.CharField(max_length=200)
+	password = models.CharField(max_length=200)
+	phone_number = models.CharField(max_length=20)
+	birthday = models.DateTimeField('date published')
```

Comment:
DateTimeField 활용 좋습니다~ 질문하신대로 사용자가 입력한 값으로 저장하고 추후에 수정을 하고 싶으시다면 아무런 설정 필요 없이 기본값으로 작성하셔도 충분합니다!
```suggestion
	birthday = models.DateTimeField()
```